scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+    paste tmp.tsv tmp.x.tsv > ${OUTPUT}.tmp
+fi
+head -1 ${OUTPUT}.tmp > tmp_header2.txt
+
+tr "\t" "\n" < tmp_header2.txt | grep -n ${SEGMENT_MEAN_COL} | cut -f1 -d: > tmp_col_num
+COL_NUM=`cat tmp_col_num`
+echo $COL_NUM
+
+cut -f$COL_NUM  ${OUTPUT}.tmp > col_data
+cut --complement -f $COL_NUM ${OUTPUT}.tmp > ${OUTPUT}.tmp_header2
+paste ${OUTPUT}.tmp_header2 col_data >${OUTPUT}
+    >>>
+
+    output {
+        File outFile="${OUTPUT}"
+    }
+
+    runtime {
+        docker : "ubuntu:16.04"
+        memory: select_first([mem_gb, 2000]) + " GB"
+        # Note that the space before SSD and HDD should be included.
+        disks: "local-disk " + select_first([disk_space_gb, 100]) + if use_ssd then " SSD" else " HDD"
+        preemptible: select_first([preemptible_attempts, 3])
+        cpu: select_first([cpu, 1])
+        bootDiskSizeGb: select_first([boot_disk_gb, 20])
+    }
+}
+
+task MergeSegmentByAnnotation {
+	File seg_file
+	Array[String] annotations
+    File ref_fasta
+    File ref_fasta_dict
+    File ref_fasta_fai
+
+	File? gatk4_jar_override
+
+	String output_name = basename(seg_file)
+
+    Int? max_merge_distance
+
+    # Runtime parameters
+    Int? mem_gb
+    String gatk_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu
+    Boolean use_ssd = false
+
+    # You may have to change the following two parameter values depending on the task requirements
+    Int default_ram_mb = 3000
+    # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).
+	Int default_disk_space_gb = 100
+
+    # Mem is in units of GB but our command and memory runtime values are in MB
+    Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+    Int command_mem = machine_mem - 1000
+
+	command <<<
+
+        set -e
+        export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
+
+		echo "======= Merging "
+		gatk --java-options "-Xmx${command_mem}m" \
+			MergeAnnotatedRegionsByAnnotation \
+            --segments ${seg_file} \
+            ${"--max-merge-distance " + max_merge_distance} \
+            --annotations-to-match ${sep=" --annotations-to-match " annotations} \
+            -O ${output_name}.merged.seg -R ${ref_fasta}
+
+	>>>
+
+    runtime {
+        docker: gatk_docker
+        memory: machine_mem + " MB"
+        # Note that the space before SSD and HDD should be included.
+        disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+        preemptible: select_first([preemptible_attempts, 3])
+        cpu: select_first([cpu, 1])
+        bootDiskSizeGb: 20
+    }
+
+    output {
+        File cnv_merged_seg = "${output_name}.merged.seg"
+    }
+}
+
+# TODO: No non-trivial heredocs in WDL.  Add this to the script directory and call via anaconda (future release)
+# TODO: This is a ham-fisted algorithm.  Better approaches exist if this does not meet needs.
```

Comment:
Again, use of "ham-fisted" through seems ham-fisted to me...better to just use some drier language.